src/pages/data_submission/DataSubmissionForm.jsx

```diff
@@ -99,9 +99,11 @@ export const DataSubmissionForm = () => {
         if (Array.isArray(files)) {
           files.forEach((file, idx) => {
             const [[duosFileType, duosFileObj]] = Object.entries(file)
-            // Consent has code to look for the string as formatted below
-            const fieldKey = `${field}[${idx}].${duosFileType}`
-            multiPartFormData.append(fieldKey, duosFileObj, duosFileObj.fileName)
+            if (duosFileObj !== null) {
```

Comment:
We should be filtering the object elsewhere so if the value is undefined we don't get into this logic, but we're not so this is the path of least resistance to fix this.